etc/WMAgentConfig.py

```diff
@@ -297,6 +297,12 @@
 # mysql*Poller sections were made optional and are defined in the
 # wmagent-mod-config file
 
+# Email alert configuration
+config.section_("EmailAlert")
+config.EmailAlert.toAddr = ["egough@cern.ch"] # additional emails can be appended to the list
```

Comment:
How about setting "cms-service-production-admins" as `contactAddr` and then assign this variable to this line and also to the L72 (config.Agent.contact)?